components/ILIAS/Export/xml/ilias_user_11.dtd

```diff
@@ -0,0 +1,160 @@
+<!ENTITY UDFDefinitions SYSTEM "ilias_udf_definition_3_10.dtd">
```

Comment:
Hi @kergomard  , we'll have to provide a new DTD for user objects since CAS is no longer a valid option in the XML document. If you have any objections, please leave a comment.